packages/slate-cssvar-loader/config.js

```diff
@@ -0,0 +1,19 @@
+const slateConfig = require('@shopify/slate-config');
+
+module.exports = slateConfig.generate({
+  items: [
+    {
+      id: 'cssVarLoaderEnable',
+      default: true,
+      description: 'Enable/disable cssvar loader plugin',
+      type: 'boolean',
+    },
+    {
+      id: 'cssVarLoaderLiquidPath',
+      default: slateConfig.resolveTheme('src/snippets/css-variables.liquid'),
```

Comment:
Can this value be an array so users can specify more than one file to pull variables from?

Reply:
Yeah that should not be a problem, on it.